src/main/java/core/basesyntax/Supply.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import dao.WareHouseDao;
+import dao.impl.WareHouseDaoImpl;
+
+public class Supply extends OperationStrategy {
```

Comment:
fix naming, strategy class should be applying handlers. Handlers don't extend strategy
call base class/interface OperationHandler/ Handler etc
and this - SupplyHandler